packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -49,6 +60,30 @@ class ClayDropdownBase extends ClayComponent {
 		this._eventHandler.removeAllListeners();
 	}
 
+	/**
+	 * Align dropdown menu.
+	 * @protected
+	 */
+	_alignAndFocusDropdown() {
+		if (this.expanded && this._alignElementSelector) {
+			let alignElement = this.element.querySelector(
+				this._alignElementSelector
+			);
+
+			let bodyElement = this._getMenuElement();
+
+			if (alignElement) {
+				this._alignedPosition = Align.align(
+					bodyElement,
+					alignElement,
+					Align.BottomLeft
+				);
+			}
+
+			bodyElement.querySelector('li a, li input').focus();
```

Comment:
hey @carloslancha we want to cover the `li button` cases too? If not, we have to add a condition if do not find anything inside.

Reply:
I think we don't have `li button` cases. Anyway, I'm extracting this to a method that could be overwritten to fit user puroposes and adding a safe condition . Thx!

Reply:
Nvm, we're not adding this feature in this way, I'm opening a new issue for this.